fabtests/Makefile.am

```diff
@@ -40,6 +40,7 @@ bin_PROGRAMS = \
 	functional/fi_resmgmt_test \
 	functional/fi_rdm_atomic \
 	functional/fi_rdm_multi_recv \
+	functional/fi_msg_multi_recv \
```

Comment:
I did not review these changes.  Is it possible to change the fi_rdm_multi_recv to support all EP types instead?  (And rename that test).

Reply:
That is more cleaner. I will do that.